### ISSUE TITLE
[Accessibilty] Action available to mouse users unavailable to keyboard users

### ISSUE THREAD

**Issue by amberhinds** [ProgressPlanner/brand-assets#13]
* [ ] I've read and understood the [contribution guidelines](https://github.com/Emilia-Capital/.github/blob/main/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened

I tested this by looking at the https://progressplanner.com/ website. When the logo is right-clicked, it triggers a modal to open. There is no way for a keyboard user to access this same functionality. This is a violation of [WCAG Success Criterion 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html).

#### Step-by-step reproduction instructions
1. Go to https://progressplanner.com/ 
2. Right click the logo to see the dialog.
3. Close the dialog.
4. Tab through the header with the Tab key and see there is no way to open the dialog. Clikcing the logo is a link to the homepage.
5. Notice there is no link to a brand assets page in the header or footer navigation (which fails [2.4.5 Multiple Ways](https://www.w3.org/WAI/WCAG22/Understanding/multiple-ways.html))

#### Expected results
All functionality of the content is operable through a keyboard interface.

Recommended fix:
1. Add a button that is hidden until it receives keyboard focus, so that users can tab to it and trigger the dialog. The button could be an icon only (the core button to open images in light boxes is a good example). The button needs these ARIA attributes:
   - `aria-label="brand assets"`
   - `aria-haspopup="dialog"`
   - `aria-controls` referencing the ID of the dialog

2. Add the brand assets link to your header or footer navigation so there are multiple ways to access it.

#### Actual results
There is no way for a keyboard-only user to trigger the brand asset dialog.

### Screenshots, screen recording, code snippet
If possible, please provide a screenshot, a screen recording or a code snippet which demonstrates the bug.

### Technical info

* If relevant, which editor is affected (or editors): 
- [ ] Block Editor
- [ ] Gutenberg Editor
- [ ] Elementor Editor
- [ ] Classic Editor 
- [ ] Other: 

* Which browser is affected (or browsers): 
- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Other: 

#### Used versions
* Device you are using:
* Operating system:
* PHP version:
* WordPress version: 
* WordPress Theme:
* Plugin version: 
* Gutenberg plugin version: 
* Elementor plugin version: 
* Classic Editor plugin version: 
* Relevant plugins in case of a bug:
